velox/exec/SortWindowBuild.cpp

```diff
@@ -165,12 +165,40 @@ void SortWindowBuild::spill() {
   data_->pool()->release();
 }
 
-void SortWindowBuild::computePartitionStartRows() {
-  partitionStartRows_.reserve(numRows_);
+// Use forward-to-back double search algorithm to find next partition start row.
```

Comment:
Would you add a link to description of the algorithm. This one seems to have a nice description: https://medium.com/@insomniocode/search-algorithm-double-front-and-back-20f5f28512e7